packages/safe-tools-client/app/components/setup-safe-modal/index.gts

```diff
@@ -39,44 +54,51 @@ export default class SetupSafeModal extends Component<Signature> {
             <li>You are the owner of the safe. Only the safe owner can schedule
               payments.
             </li>
-            <li>Once you have scheduled payments, the module (source code here)  {{! TODO: Add link }}
+            <li>Once you have scheduled payments, the module
+              <a
+                href='https://github.com/cardstack/cardstack-module-scheduled-payment'
+                target='_blank'
+                rel='external'
+              >
+                (source code here)
+              </a>
```

Comment:
I see that the design has it this way, I wonder whether we should make it visible as a link though 🤔 like via text colour or permanent underline.

Reply:
Agree that it should be visible as a link! I can work on this in a separate PR.